src/_tutorials/language/futures.md

```diff
@@ -11,22 +11,41 @@ nextpage:
 <div class="mini-toc" markdown="1">
   <h4>What's the point?</h4>
 
-  * Dart is single-threaded.
-  * Synchronous code can make your program freeze.
-  * Use `Future` objects (_futures_) to perform asynchronous operations.
-  * Use `await` in an async function to suspend execution until a future completes.
-  * Or use the `then()` method.
-  * Use try-catch expressions in async functions to catch errors.
-  * Or use the `catchError()` method.
-  * You can chain futures to run asynchronous functions in order.
+  * Dart code runs straight through, in a single "thread" of execution.
+  * Code that blocks the thread of execution can make your program freeze.
+  * `Future` objects (_futures_) represent asynchronous operations —
+    processing or I/O to be completed later.
+  * To suspend execution until a future completes,
+    use `await` in an async function (or use `then()`).
+  * To catch errors, use try-catch expressions in async functions
+    (or use `catchError()`).
+  * To run code concurrently, create an _isolate_
+    (or for web apps, a _worker_).
 </div>
 
-Dart is a single-threaded programming language. If any code blocks the thread
-of execution (for example, by performing a long-running calculation or
-blocking on I/O) the program effectively freezes. Asynchronous operations let
+Dart code runs in a predictable order, in a single "thread" of execution.
+Any code that blocks that thread of execution
+(for example, by performing a long-running calculation or blocking on I/O)
+can freeze a Dart program. Asynchronous operations let
 your program complete other work while waiting for an operation to complete.
 Dart uses `Future` objects to represent asynchronous operations.
 
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  All Dart code runs in the context of an **isolate** that
+  owns all of the memory that the Dart code uses.
+  While Dart code is executing, no other code in the same isolate can run.
+  This page describes how to pause execution and
```

Comment:
"pause" in relation to isolates usually means something else.
Maybe "delay" (since "yield" also means something else, all the good names are taken!)